shoop/admin/modules/orders/static_src/create/reducers/lines.js

```diff
@@ -54,15 +54,15 @@ function getDiscountsAndTotal(quantity, baseUnitPrice, unitPrice, updateUnitPric
     if (updateUnitPrice) {
         updates.unitPrice = unitPrice;
     }
-    var totalBeforeDiscount = baseUnitPrice * quantity;
+    var totalBeforeDiscount = (baseUnitPrice * quantity).toFixed(2);
     var total = +(unitPrice * quantity).toFixed(2);
     updates.total = total;
     if (baseUnitPrice < unitPrice || unitPrice < 0) {
         updates.discountPercent = 0;
         updates.discountAmount = 0;
         return updates;
     }
-    var discountAmount = totalBeforeDiscount - total;
+    var discountAmount = (totalBeforeDiscount - total).toFixed(2);
```

Comment:
Ok, so. Yes it works if we remove the rounding hfrom JS all together (from lines). That just leads to a lot of decimal points that might confuse merchant. It seems that this is our compromise now based on your other comment.

Also yes the order total at the bottom of editor is using 2 decimals already which I am going leave like it is.